app/views/fields/belongs_to/_form.html.erb

```diff
@@ -17,9 +17,6 @@ that displays all possible records to associate with.
 %>
 
 <%= f.label field.permitted_attribute %>
-<%= f.collection_select(
-  field.permitted_attribute,
-  field.candidate_records,
-  :id,
-  :to_s,
-) %>
+<%= f.select(field.permitted_attribute) do %>
+  <%= options_for_select(field.candidate_options) %>
```

Comment:
yep - I was using `option` in the sense of the `<option>` tag for `<select>` fields, but I see what you mean. I think I like `associated_resource_options`.